internal/configs/virtualserver.go

```diff
@@ -988,6 +992,72 @@ func (p *policiesCfg) addWAFConfig(
 	return res
 }
 
+func (p *policiesCfg) addBadosConfig(
```

Comment:
Could we have a test similar to TestAddWafConfig?

Reply:
Add uni test TestAddBadosConfig